lib/ex_doc/retriever.ex

```diff
@@ -263,10 +263,11 @@ defmodule ExDoc.Retriever do
   defp get_defaults(signature, name, arity) do
     case Enum.count(signature, &match?({:\\, _, [_, _]}, &1)) do
       0 -> []
-      defaults -> for default <- (arity-defaults..arity-1), do: "#{name}/#{default}"
+      defaults -> for default <- (arity - defaults..arity - 1), do: "#{name}/#{default}"
```

Comment:
What do you think of `(arity - defaults)..(arity - 1)`. The current way reads weird. :)